### DISCUSSION TITLE
AWS Internet facing ALB 2.7.2 + Istio Ingress 1.21.2 not working

### DISCUSSION THREAD

**Discussion by nphariharan** [istio/istio#51194] [Q&A]
I am trying to setup the below in AWS EKS Cluster 1.29

AWS Internet facing Application Load Balancer 2.7.2 + Istio Ingress 1.21.2. 

[yamls.zip](https://github.com/istio/istio/files/15402999/yamls.zip)
My service is not responding and throwing 502 Bad Gateway error.

istio-Ingress-gateway - throwing "NR filter_chain_not_found"- 

Attached my yaml files for reference.

Please kindly help to resolve this issue.

Thanks,
Hariharan.

**Comment by Govind10g** [comment]
Can you check your application log as well where istio is forwarding the request.
Is it healthy and running fine ?
